Description: Disable timestamp generation in doxygen
 If Doxyfile contains HTML_TIMESTAMP = YES, Doxygen will
 add a timestamp to its generated documentation. To allow
 reproducible builds this should be disabled.
Author: Vasudev Kamath <kamathvasudev@gmail.com>
Forwarded: no
Last-Update: 2016-03-25

--- a/doc/Doxyfile.in
+++ b/doc/Doxyfile.in
@@ -1129,7 +1129,7 @@
 # The default value is: YES.
 # This tag requires that the tag GENERATE_HTML is set to YES.
 
-HTML_TIMESTAMP         = YES
+HTML_TIMESTAMP         = NO
 
 # If the HTML_DYNAMIC_SECTIONS tag is set to YES then the generated HTML
 # documentation will contain sections that can be hidden and shown after the
